src/main/java/org/snomed/snowstorm/core/data/services/servicehook/CommitServiceHookClient.java

```diff
@@ -54,6 +55,15 @@ public void preCommitCompletion(Commit commit) throws IllegalStateException {
 			logger.error("Commit service hook failed for branch {}, commit {}, url {}, cookie {}",
 					commit.getBranch().getPath(), commit.getTimepoint().getTime(), serviceUrl,
 					obfuscateToken(authenticationToken), e);
+			// e will be HttpClientErrorException.Conflict if review for branch is incomplete.
+			// Regardless, if the commit is a promotion, we will always abort if an exception is encountered.
+			boolean promotion = commit.getCommitType().equals(Commit.CommitType.PROMOTION);
+			if (promotion) {
```

Comment:
Checking the HTTP status here would prevent user/devops misdirection if the AAG service is offline.